soak-shiftplanning.py

```diff
@@ -8,13 +8,15 @@
 parser.add_argument('teilnehmer')
 parser.add_argument('--seed', type=int, default=0)
 
-num_workers = 3
```

Comment:
Maybe

```
num_workers = defaultdict(lambda: 3)
num_workers['cleaning'] = 4
```

And looking up num_workers by task would be more elegant.

Reply:
Covered by following commit